File: 1-intro-estruturas-selecao-repeticao/g-sum.cpp
#include <iostream>

int main() {
    int t;
    std::cin >> t;  // Número de casos de teste

    // Processa cada caso de teste
    while (t--) {
        int a, b, c;
        std::cin >> a >> b >> c;

        // Verifica se algum número é a soma dos outros dois
        // Testa todas as possibilidades: a+b=c, b+c=a, a+c=b
        if (a + b == c || b + c == a || a + c == b) {
            std::cout << "YES" << std::endl;
        } else {
            std::cout << "NO" << std::endl;
        }
    }

    return 0;
}

File: 01-intro-estruturas-selecao-repeticao/g-sum.py
# Lê o número de casos de teste
t = int(input())

# Processa cada caso de teste
for _ in range(t):
    # Lê os três números como inteiros
    a, b, c = map(int, input().split())
    
    # Verifica se algum número é a soma dos outros dois
    # Testa todas as possibilidades: a+b=c, b+c=a, a+c=b
    if a + b == c or b + c == a or a + c == b:
        print("YES")
    else:
        print("NO")